src/Network/Transport/TCP.hs

```diff
@@ -861,8 +878,8 @@ apiCloseEndPoint transport evs ourEndPoint =
 -- the transport down. We must be careful to close the socket when a (possibly
```

Comment:
This comment is outdated. When is the thread running `handleConnectionRequest` terminated?